CSS/components/modal.css

```diff
@@ -0,0 +1,39 @@
+.dialog-main{
+    display: none;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
+    top: 0;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.5);
+    width: 100vw;
+    height: 100vh;
+}
+.open{
+    display: flex;
```

Comment:
Use until class for this, like d-flex since this is very frequently used

Reply:
here I am having a specific usage for this class in modal with js to open the modal but will also add flex general class in utils 
Thank you for the suggestion Added a general class flex in util